src/main/scala/diplomacy/Nodes.scala

```diff
@@ -248,6 +415,13 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   val inward = this
   val outward = this
 
+  /**
+   * Given counts of known inward and outward binding and inward and outward
+   * star bindings, return the resolved inward stars and outward stars.
```

Comment:
what is a "star binding"? We never talked about it. Not sure if that was intentional vs running out of time to type :)

Reply:
This was a running out of time (or really, the passing out due to being awake for 20 hours) thing. This is where I got to when I hit the "Understanding NodeBindings and running into a recursive block" described above.